### DISCUSSION TITLE
Integration: Webhook

### DISCUSSION THREAD

**Discussion by MGkBIZ** [wpappointments/wpappointments#227] [Ideas]
webhook triggers:
create meeting, modify, end.

**Comment by MGkBIZ** [comment]
this will allow integration with make and zapier (smsapi, mailerlite, freshmail, ticket system, etc.)

**Comment by wppoland** [comment]
Thanks, and apologies for the late reply. That is exactly the intent: webhooks will fire on all appointment lifecycle events, so Make, Zapier and similar services can pick them up for SMS, email or ticketing without a dedicated integration for each. Webhooks are planned after v1.0 of the free core plugin, which is the current focus. Related discussion: #56.
